gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua

```diff
@@ -664,14 +665,14 @@ function SKIN:PaintHelpLabelTTT2(panel, w, h)
 	drawBox(0, 0, w, h, colors.helpBox)
 	drawBox(0, 0, 4, h, colors.helpBar)
 
-	local textTranslated = TryT(panel:GetText())
+	local textTranslated = ParT(panel:GetText(), panel:GetParams())
 	local textWrapped = drawGetWrappedText(
 		textTranslated,
 		w - 2 * panel.paddingX,
 		panel:GetFont()
 	)
 
-	local _, heightText = drawGetTextSize(textTranslated, panel:GetFont())
+	local _, heightText = drawGetTextSize("", panel:GetFont())
```

Comment:
Explanation? Probably as a in-code comment

Reply:
whyt should be explained here?

Reply:
Why using an empty string?

Reply:
Because it doesn't change anything (only the height is relevant)
But in another part of the code I had to use an empty string because and the time of the calculation, the text is not yet known. I therefore changed it here as well to keep it consistent.

Reply:
Theoretically yes, but the width differs as well regarding e.g. `w` and `i`. So if you are sure, I gonna close this.

Reply:
As I'm only using the height, this does not matter. I veriefied it for the texts used in the vgui rework and they all looked good to me.